client/src/components/question/ThirdPartyIcons.js

```diff
@@ -38,12 +38,15 @@ class ThirdPartyIcons extends React.Component {
         if (index !== null) {
           hiddenId += '-' + index;
         }
-        let answer = document.querySelector(hiddenId);
+		//answer is a span with innerHTML set to the value we're interested in
```

Comment:
Do you mind changing these comments so that the line breaks match up, and there's a space between the start of the comment mark and the actual comment text? Thanks! (See overall review comment for more about why I'm not just pulling and doing it myself.)